node-api/handlers/beacon/types/response.go

```diff
@@ -141,3 +142,28 @@ type Sidecar struct {
 type SidecarsResponse struct {
 	Data []*Sidecar `json:"data"`
 }
+
+// PendingPartialWithdrawalsResponse has a version field to indicate the fork version.
+// https://ethereum.github.io/beacon-APIs/#/Beacon/getPendingPartialWithdrawals
+type PendingPartialWithdrawalsResponse struct {
+	Version string `json:"version"`
```

Comment:
isn't it kind of weird that we only have version over this specific response? Why others do no need Version?

Reply:
I see version in few other APIs as well like https://ethereum.github.io/beacon-APIs/#/Beacon/getPendingDeposits, seems it got introduced in newer APIs. One reason could be because the API will error out if state is fetched before electra.